Doc/using/cmdline.rst

```diff
@@ -551,7 +551,9 @@ Miscellaneous options
      *n* must be greater than or equal to 1.
      This option may be useful for users who need to limit CPU resources of a
      container system. See also :envvar:`PYTHON_CPU_COUNT`.
-     If *n* is ``default``, nothing is overridden.
+     If *n* is ``default``, nothing is overridden, if *n* is ``process``,
+     :func:`os.os.process_cpu_count` becomes an alias of :func:`os.cpu_count`
```

Comment:
That's wrong. It's the opposite: os.cpu_count becomes an alias to os.process_cpu_count. No?

I don't see why you mention multiprocessing.cpu_count(). -X cpu_count=process has no effect on multiprocessing.cpu_count(), since multiprocessing.cpu_count() calls os.process_cpu_count(). No?

Reply:
Ah got it, I missed